bigtable_input.py

```diff
@@ -75,6 +76,11 @@
 _game_row_key = re.compile(r'g_(\d+)_m_(\d+)')
 _game_from_counter = re.compile(r'ct_(\d+)_')
 
+# To parse out project, instance, and table from a Bigtable path like:
+#   'projects/tensor-go/instances/minigo-instance/tables/v14-games'
+_unpack_table_path = re.compile(
+    r'^projects/([^/]+)/instances/([^/]+)/tables/([^/]+)$')
```

Comment:
I see bigtable is a very stringly typed API

Reply:
Yes. The `BigtableClient` doesn't pickle, so it can't be passed to via `multiprocessing.map`. I needed to pass the three parameters required to create the client (project, instance, table), but the client doesn't have inspectors for those. It did have a `name` attribute, in the above format.

The alternative would be for me to have `GameQueue` save these parameters and pass them directly. This might be safer, now that I think about it, because the API doesn't describe the format of the `name` attribute in its docs. It would still be a three-string triple, but I can make it a named tuple, which would help.